doc/ipythonconsole.rst

```diff
@@ -24,11 +20,9 @@ Each console implements a robust two-process IPython session, with a lightweight
 You can also connect to external local and remote kernels, including those managed by QtConsole sessions or the Jupyter Notebook, through the :guilabel:`Connect to an existing kernel` dialog under the same menus
```

Comment:
```suggestion
You can also connect to external local and remote kernels, including those managed by QtConsole sessions or the Jupyter Notebook, through the :guilabel:`Connect to an existing kernel` dialog under the same menu.
```

I think this is a syntax error.

Reply:
```suggestion
You can also connect to external local and remote kernels, including those managed by QtConsole sessions or the Jupyter Notebook, through the :guilabel:`Connect to an existing kernel` dialog under the same menus.
```

English syntax, not reST syntax, but sure. Also, menus needs to be plural since its referencing multiple (see the previous sentence).

Reply:
Yes, it's an English error. And I also agree with your suggestion.